### PR TITLE
Reconfigure client settings

### PR DESCRIPTION
**Pull request by ivoanjo**
Increasing attempt and call timeouts. Removing override of other defaults.

(I'm opening this PR on behalf of Mohammed, since he hasn't yet done the github setup)

### CODE REVIEW

**Review by ivoanjo — COMMENTED** [review]
I can't approve this PR since I opened it, but change looks reasonable.
CI is failing -- we should look into that before merging.

**Review by gimki — APPROVED** [review]

**Pull request merged**
